model_analyzer/model_manager.py

```diff
@@ -276,7 +275,11 @@ def _get_measurement_if_config_duplicate(self, run_config):
         if model_config_name not in results[model_name]:
             return False
         measurements = results[model_name][model_config_name][1]
-        if perf_config_str in measurements:
+
+        # For backward compatibility
```

Comment:
Can you elaborate on the backward computability that is being covered here?

Reply:
Resolved in comment